App.js

```diff
@@ -160,32 +159,8 @@ class App extends React.Component<Props, *> {
     const env = await setupEnv();
     log.info('Environment: ', env);
     this.setState({ env });
-    /**
-     * First, we need to set the defaults for Remote Config.
-     * This makes the default values immedidately available
-     * on app load and can be used.
-     *
-     * @url https://rnfirebase.io/reference/remote-config#setDefaults
-     */
-
-    remoteConfig()
-      .setDefaults(INITIAL_FEATURE_FLAGS)
-      .then(() => log.info('Firebase Config: Defaults loaded and available.'))
-      .catch(e => log.error('Firebase Config: An error occured loading defaults:', e));
-
-    /**
-     * Secondly, we need to activate any remotely fetched values
-     * if they exist at all. The values that have been fetched
-     * and activated override the default values above (see @url
-     * above).
-     *
-     * @url https://rnfirebase.io/reference/remote-config#activate
-     */
-
-    remoteConfig()
-      .activate()
-      .then((r) => log.info('Firebase Config: Activation result was:', r))
-      .catch(e => log.error('Firebase Config: An error occured loading defaults:', e));
+
+    await initFirebase();
```

Comment:
Did we mean to `await` initFirebase here? Just some context: when we added feature we intended it to be completely non-blocking especially in the `App.js` file. I also checked `initFirebase` method in `src/services/firebase.js` and it appear that whilst it is an `async` function - we don't return a value or `await` anything in the method...

Reply:
I've reverted the change and I'm waiting of only default values to be set now. 
this means that if there are no baseUrl (did not fetch it in time, no cached value), local translations are used.
Later, when remote config values are fetched, translations are updated if needed.